velox/vector/tests/utils/VectorTestBase.h

```diff
@@ -129,9 +129,13 @@ class VectorTestBase {
     return vectorMaker_.rowVector(rowType, size);
   }
 
-  std::vector<RowVectorPtr>
-  createVectors(const RowTypePtr& type, size_t vectorSize, uint64_t byteSize) {
-    VectorFuzzer fuzzer({.vectorSize = vectorSize}, pool());
+  std::vector<RowVectorPtr> createVectors(
+      const RowTypePtr& type,
+      size_t vectorSize,
+      uint64_t byteSize,
+      VectorFuzzer::Options fuzzerOpts = {}) {
```

Comment:
If we pass fuzzerOpts, then do we still need to pass vectorSize? Thanks!

Reply:
I've refactored this API and split it into two APIs.
```C++
std::vector<RowVectorPtr> createVectors(
      const RowTypePtr& type,
      uint64_t byteSize,
      const VectorFuzzer::Options& fuzzerOpts);

std::vector<RowVectorPtr>
  createVectors(const RowTypePtr& type, size_t vectorSize, uint64_t byteSize);
```